### PR TITLE
Metal3-io-2022-Annual Review

### PR DESCRIPTION
**Pull request by Rozzii**

### CODE REVIEW

File: reviews/2022-metal3-io-annual.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
+# Metal3-io 2022 Annual Review
+
+## Metrics
+
+This [DevStats](https://metal3.devstats.cncf.io/d/8/dashboards?orgId=1&from=now-1y&to=now-1h&editPanel=2)
+graph shows that we have a fairly consistent number of contributions
+throughout the year.
+
+### Downloads per repository:
```
Comment (tuminoid) [inline reviews/2022-metal3-io-annual.md:9]: Header does not need colon in the end

File: reviews/2022-metal3-io-annual.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+## Adoption
+
+The project's adopters list can be found [here](https://github.com/metal3-io/metal3-docs/blob/main/ADOPTERS.md).
+
+**Adopters:**
+
+**Companies that are using Metal3 in production:**
+
+`IKEA IT AB`: "IKEA IT AB uses Metal3 to
```
Comment (tuminoid) [inline reviews/2022-metal3-io-annual.md:66]: Weird line wrapping

File: reviews/2022-metal3-io-annual.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+found [here](https://github.com/metal3-io/metal3-docs/blob/master/maintainers/ALL-OWNERS).
+We have introduced the option of moving people to emeritus approvers and
+reviewers list for people who became less active in the review/maintenance
+process.
+
+## Adoption
+
+The project's adopters list can be found [here](https://github.com/metal3-io/metal3-docs/blob/main/ADOPTERS.md).
+
+**Adopters:**
```
Comment (tuminoid) [inline reviews/2022-metal3-io-annual.md:62]: This seems duplicate as there is header just above, and its followed by equally strong line.

File: reviews/2022-metal3-io-annual.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+     - or providing with Metal3-io with any other opportunity to improve its
+       traction/visibility
+
+
+## Security processes
+
+The Metal3-io project's general security policy is located [here](https://metal3io.netlify.app/security_policy.html).
+
+In addition to the project's security policy, all the active repositories
+have a SECURITY_CONTACTS file. The SECURITY_CONTACTS file informs
```
Comment (tuminoid) [inline reviews/2022-metal3-io-annual.md:227]: Filenames should go inside `ticks`.

File: reviews/2022-metal3-io-annual.md [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+  - the project's security policy
+  - basic rules of security vulnerability disclosure
+
+Optionally the SECURITY_CONTACTS may contain:
+  - repository specific security fix back porting information
+  - repository specific security fix release information
+
+## Incubation
+
+We think we meet the criteria of incubation. We have shown healthy number of
```
Comment (tuminoid) [inline reviews/2022-metal3-io-annual.md:238]: We think Metal3 (or Metal3-io) meets ...

File: reviews/2022-metal3-io-annual.md (hunk 2 above)
Comment (Rozzii) [reply reviews/2022-metal3-io-annual.md]: Thanks, done.

File: reviews/2022-metal3-io-annual.md (hunk 3 above)
Comment (Rozzii) [reply reviews/2022-metal3-io-annual.md]: Thanks, done.

File: reviews/2022-metal3-io-annual.md (hunk 4 above)
Comment (Rozzii) [reply reviews/2022-metal3-io-annual.md]: Thanks, done.

File: reviews/2022-metal3-io-annual.md (hunk 5 above)
Comment (Rozzii) [reply reviews/2022-metal3-io-annual.md]: Thanks, done.

File: reviews/2022-metal3-io-annual.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+    separate tutorials that will showcase the deployment process on bare metal
+    environment that is as close to a production environment as possible.
+
+  - In 2022 certain aspects of Metal3 security has been improved, but there
+    is more planned specially related to the topics of container security
+    and related to the testing of Metal3 security features during e2e tests.
+
+  - There are improvements done on the CI infrastructure of the Metal3
+    project, but there are more to come in terms of securing and speeding the
+    test processes to help contributors receive feedback faster.
```
Comment (dtantsur) [inline reviews/2022-metal3-io-annual.md:190]: Really late from me (sorry!), but in a similar vain, there is an effort to run Metal3 CI on Ironic upstream: https://review.opendev.org/c/openstack/ironic/+/863873

Comment (Rozzii) [reply reviews/2022-metal3-io-annual.md:190]: Added, thanks for the info @dtantsur !

**Comment by Priyankasaggu11929** [comment]
Hey all :wave:, I'll be reviewing the annual report. Thanks!

**Comment by raravena80** [comment]
@Priyankasaggu11929 following up with the review. Were you able to complete it?

Thanks!

**Comment by raravena80** [comment]
Here's the [doc link](https://docs.google.com/document/d/14KObDX-ybKzgZ6sjpfFXNyNOmeWIoIdfd7tCtzwVNhk/edit#heading=h.3ff1v7xs75xa) to the review.

**Comment by TheFoxAtWork** [comment]
@nikhita @RichiH @cathyhongzhang please review the linked comment/recommendation from the TAG and provide concurrence/approval.

**Comment by Priyankasaggu11929** [comment]
>@Priyankasaggu11929 following up with the review. Were you able to complete it?

I was out last week. Just getting back to the notification now. :slightly_smiling_face: 

> Here's the [doc link](https://docs.google.com/document/d/14KObDX-ybKzgZ6sjpfFXNyNOmeWIoIdfd7tCtzwVNhk/edit#heading=h.3ff1v7xs75xa) to the review.

Yes, :point_up: is the correct document.

**Comment by amye** [comment]
Marking this as complete, annual reviews are no longer a requirement for sandbox. 
We'll be rolling out more efficient ways to show project health and metrics in 2024.

**Pull request merged**